apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_producer.erl

```diff
@@ -591,8 +653,10 @@ error_summary(Map, [Error]) ->
 error_summary(Map, [Error | More]) ->
     Map#{first_error => Error, total_errors => length(More) + 1}.
 
-check_if_healthy_leaders(ClientId, ClientPid, KafkaTopic, MaxPartitions) when is_pid(ClientPid) ->
-    case wolff_client:get_leader_connections(ClientPid, KafkaTopic, MaxPartitions) of
+check_if_healthy_leaders(ActionResId, ClientId, ClientPid, KafkaTopic, MaxPartitions) when
+    is_pid(ClientPid)
+->
+    case wolff_client:get_leader_connections(ClientPid, ActionResId, KafkaTopic, MaxPartitions) of
```

Comment:
how does it behave if it's a template ?

Reply:
Since this is only called in `check_topic_and_leader_connections/4`, and we have the `maybe` clause guarding this call, it won't be called if the topic is a template.